config/config.yaml

```diff
@@ -57,8 +57,7 @@ dependencies:
         host: localhost
         port: 8080
     notify-service:
-        api_key: notify_api_key
-        service_id: sdc_service_id
+        url: "http://notifygatewaysvc-dev.apps.devtest.onsclofo.uk/emails/"
```

Comment:
We default to a local version on all other services, any reason why we are using dev here?